src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -35,6 +35,7 @@ export class ApiInterceptorService implements HttpInterceptor {
           case 404:
             this.router.navigate(['/404']);
             break;
+          case 400:
           case 409:
```

Comment:
You can remove this line, since we don't use `409` anymore.

Reply:
fixed.